### PR TITLE
Add ucs_query

### PR DESCRIPTION
**Pull request by movinalot**
##### SUMMARY

Cisco UCS Module for performing class and dn based queries

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

ucs_query

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Review by dsoper2 — APPROVED** [review]
shipit

File: lib/ansible/modules/remote_management/ucs/ucs_query.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        default: localhost
+        type: str
+
+requirements:
+    - ucsmsdk
+
+author:
+    - John McDonough (@movinalot)
+    - CiscoUcs (@CiscoUcs)
+version_added: "2.9"
```
Comment (gundalow) [inline lib/ansible/modules/remote_management/ucs/ucs_query.py]:
```suggestion
version_added: '2.10'
```

**Review by gundalow — COMMENTED** [review]
Just one minor change then we can get this merged.

**Comment by gundalow** [comment]
@movinalot Thanks for the quick fix
rebuild_merge

**Comment by movinalot** [comment]
@gundalow Thank you!

**Pull request merged**